float64[] height
float64[] radius
float64 container_height_m
